nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -193,10 +199,10 @@ def _get_input_scale_shape(
         """
 
         is_weights = target_point.is_weight_target_point()
+        input_shape = nncf_graph.get_input_shape_for_insertion_point(target_point)
```

Comment:
As far as understand, there are input shapes in edges of nncf graph. However, constant nodes are not added to the nncf graph for PyTorch. Could you explain how it work for weights?

please, also remove it from the else block: 
```
        else:
            input_shape = nncf_graph.get_input_shape_for_insertion_point(target_point)
            channel_axes = (1,)  # channel dim for activations
```